LandLord-legacy/src/main/java/biz/princeps/landlord/manager/WorldGuardManager.java

```diff
@@ -178,29 +185,33 @@ public void run() {
     @Override
     public boolean canClaim(Player player, Chunk currChunk) {
         RegionManager regionManager = getRegionManager(player.getWorld());
-        if (regionManager != null) {
-            int x = currChunk.getX() << 4;
-            int z = currChunk.getZ() << 4;
-            Vector v1 = new Location(currChunk.getWorld(), x, 0, z).toVector();
-            Vector v2 = new Location(currChunk.getWorld(), x + 15, 255, z + 15).toVector();
-
-            ProtectedRegion check = new ProtectedCuboidRegion("check",
-                    new BlockVector(v1.getX(), v1.getY(), v1.getZ()),
-                    new BlockVector(v2.getX(), v2.getY(), v2.getZ()));
-            List<ProtectedRegion> intersects = check
-                    .getIntersectingRegions(new ArrayList<>(regionManager.getRegions().values()));
-            for (ProtectedRegion intersect : intersects) {
-                if (!regionManager.getApplicableRegions(intersect).isMemberOfAll(wgPlugin.wrapPlayer(player))) {
-                    return false;
-                }
+        if (regionManager == null) {
+            return false;
+        }
```

Comment:
Note that this is a change of behavior. The method returned `true` before if the regionManager was `null`.

Reply:
Indeed, but it was not the expected behavior.